src/script/cpatch

```diff
@@ -47,6 +47,10 @@ while [ -n "$1" ]; do
 	    shift
 	    target="$1"
 	    ;;
+    --noall)
```

Comment:
please document what `--noall` is for in `usage()`.

Reply:
also, this change is not reflected in the commit message.

Reply:
ack, thanks!